.github/workflows/validate-workflows.yml

```diff
@@ -0,0 +1,33 @@
+name: Validate GitHub Actions Workflows
+
+on:
+  pull_request:
+    paths:
+      - "**/*.ya?ml"
+  push:
+    paths:
+      - "**/*.ya?ml"
+  # Scheduled trigger to catch workflow failure resulting from changes to the JSON schema
+  schedule:
+    # run every Saturday at 3 AM UTC
+    - cron: "0 3 * * 6"
+  # See: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_dispatch
+  workflow_dispatch:
+  # See: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#repository_dispatch
+  repository_dispatch:
+
+jobs:
+  validate:
+    runs-on: ubuntu-20.04
+    
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v2
+
+      - name: Install JSON schema validator
+        run: sudo npm install --global ajv-cli
```

Comment:
Can we avoid using sudo here? Just seems like bad practice. You could also get rid of the separate install step entirely by using `npx` to download and run together like 
```
npx -p ajv-cli ajv -s "./.github/schemas/github-action.json" -d "**/*.{yml,yaml}"
```

Reply:
I don't think it is a good idea to do two things at the same time, instead I'd be explicit on every step which makes them reproducible.

Reply:
I'm fine either way, but there's no change in reproducibility here - `npx` is the standard way of executing node commands to avoid global installs for exactly this type of scenario: https://nodejs.dev/learn/the-npx-nodejs-package-runner

The point about reproducibility is good, and what both variants share right now is not pinning the version of ajv-cli. So I'd recommend doing that in either approach
```
npm install --global ajv-cli@4.0.1
// OR
npx -p ajv-cli@4.0.1 ajv -s "./.github/schemas/github-action.json" -d "**/*.{yml,yaml}"
```

Reply:
addressed

Reply:
install is failing without Sudo.

Reply:
worked with npx

Reply:
Works for me as long as @jcchavezs is good with it.